pkg/scanners/helm/test/scanner_test.go

```diff
@@ -47,7 +47,7 @@ func Test_helm_scanner_with_archive(t *testing.T) {
 		require.NotNil(t, results)
 
 		failed := results.GetFailed()
-		assert.Equal(t, 26, len(failed))
+		assert.Equal(t, 25, len(failed))
```

Comment:
how come we lost one?

Reply:
i fixed the rego getting run twice so the number was different

Reply:
i had a bug where if you had 2 rules called `deny` it would run them both twice